tests/modules/images/webp-uploads/webp-uploads-test.php

```diff
@@ -3,9 +3,12 @@
  * Tests for webp-uploads module.
  *
  * @package performance-lab
+ * @group webp-uploads
```

Comment:
Adding this enables running just these tests locally

Reply:
Great! Maybe worth adding such an annotation to the other module test class(es) as well? Doesn't really belong here, but might make sense to just add.